sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,575 @@
+* **Other initiatives as identified by the community and Treasury Committee:** A portion of the Endowment will remain flexible to address emerging needs and support innovative ideas that are identified by the community.
+
+### **Endowment Creation & Specifications**
+
+The endowment will be created primarily via new emissions from the Stacks network over the next five years. This emissions design was created to ensure that the new Endowment had sufficient resources to fund the necessary activities and build a healthy endowment base, all while maintaining a lower overall inflation rate than the majority of Top 50 crypto ecosystems (which currently have a median of \~10%, further details are below).
+
+To fund the initial working capital, such as that needed for deployment in liquidity operations, applications, BD allocations, and similar,
+
+1. 25M STX (or equivalent other currencies) transfers from the Stacks Foundation, Hiro, Muneeb Ali, Stacks Asia Foundation, Bitcoin L2 Labs, and others. These are existing tokens and do not result in new emissions.  
+2. 100M STX will be created via a one-time token mint, immediately upon implementation of the SIP \- while immediately unlocked and usable, these tokens will be used for working capital and will not be released to the open market.
+
+The rest of the Endowment will be created via new emissions over a 60-month period.
+
+Of these new emissions,
+
+1. The first 100M STX will be minted as locked and will unlock over the first 24 months with 1/24th unlocking each month \- these tokens will also not come to the open market but rather will be sold while locked via OTC/private placements with 12-36 month lockups in order to raise the initial USD operational capital for the Endowment.  
+2. The majority (300M STX) of these new emissions will be emitted over the course of 60 months at each PoX tenure anchor block. These new tokens will be minted and deposited directly to the Treasury address (similar to, but distinct from, coinbase rewards). The specific schedule is:  
+   1. **Months 1-11:** 475 STX per anchor block  
+   2. **Months 12-23:** 1,140 STX per anchor block  
+   3. **Months 24-35:** 1,705 STX per anchor block  
+   4. **Months 36-47:** 1,305 STX per anchor block  
+   5. **Months 48-60:** 1,155 STX per anchor block
+
+The Endowment will maintain and publish a primary mint address that all funds will flow to before being deployed. There are potential options to further sustain the treasury that are not in scope of this SIP via protocol revenue paths that can be considered separately, but there are not further treasury mints beyond this endowment. See [Stacks Economic Model](https://forum.stacks.org/t/stacks-economic-model-unlocking-bitcoin-capital-long-term-growth/18035) for more details.
```

Comment:
> The Endowment will maintain and publish a primary mint address 

This address will be a smart contract address, right?  Asking because the keys controlling the funds are expected to rotate.

Reply:
If it has been discussed, I haven't been party to it.

I'll address this in a PR-level comment.

Reply:
can confirm it has been discussed, and the proposed solution should be added here (with the inclusion of the technical CAB to evaluate).

Reply:
tech specs have been added and are bring worked though. resolving this